### DISCUSSION TITLE
Puppeteer Issue

### DISCUSSION THREAD

**Discussion by annaficek** [thegetty/quire#903] [Get Help]
Hi all, 

I'm trying to install Quire and I'm having a problem with Puppeteer. Could anyone help figure out how to proceed with the below error messages? 

Thank you so much! 

MacBook-Air-7:~ agnieszkaannaficek$ npm install --global @thegetty/quire-cli

npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead
npm WARN deprecated @babel/polyfill@7.12.1: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
npm WARN deprecated puppeteer@20.9.0: < 21.5.0 is no longer supported
npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/puppeteer
npm ERR! command failed
npm ERR! command sh -c node install.js
npm ERR! ERROR: Failed to set up Chrome r115.0.5790.98! Set "PUPPETEER_SKIP_DOWNLOAD" env variable to skip download.
npm ERR! Error: Download failed: server returned code 500. URL: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/115.0.5790.98/mac-arm64/chrome-mac-arm64.zip
npm ERR!     at /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/@puppeteer/browsers/lib/cjs/httpUtil.js:100:31
npm ERR!     at ClientRequest.requestCallback (/usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/@puppeteer/browsers/lib/cjs/httpUtil.js:77:13)
npm ERR!     at Object.onceWrapper (node:events:634:26)
npm ERR!     at ClientRequest.emit (node:events:519:28)
npm ERR!     at HTTPParser.parserOnIncomingClient (node:_http_client:690:27)
npm ERR!     at HTTPParser.parserOnHeadersComplete (node:_http_common:119:17)
npm ERR!     at TLSSocket.socketOnData (node:_http_client:532:22)
npm ERR!     at TLSSocket.emit (node:events:519:28)
npm ERR!     at addChunk (node:internal/streams/readable:559:12)
npm ERR!     at readableAddChunkPushByteMode (node:internal/streams/readable:510:3)

npm ERR! A complete log of this run can be found in: /Users/agnieszkaannaficek/.npm/_logs/2024-02-22T19_32_29_091Z-debug-0.log

**Comment by Erin-Cecele** [comment]
Hi @annaficek. Can you trying installing Quire again. It looks like whatever was causing the issue may have resolved itself. Please let us know if you can get it to work. (cc'ing @fchasen for updates.)

**Comment by fchasen** [comment]
I wasn't able to reproduce it locally, but it seems like the downloads for that version of Chrome the `puppeteer` library used might have been removed. (but I think they were added back?)

The issue is likely from the `20.9.0` version being deprecated, so I'll have to update that dependency and bump all the versions to reliably fix this.

**Reply by Erin-Cecele** [reply]
Just as an update @annaficek did get install to work this time. Perhaps that's the result of adding the puppeteer library back, as you suggest above.
